standalone/detectors/fasttrack/include/fasttrack.h

```diff
@@ -50,66 +59,372 @@ class Fasttrack : public Detector {
   using c_alloc = std::allocator<X>;
 #endif
 
+  explicit Fasttrack() = default;
+
  private:
+  /// holds the callback address to report a race to the drace-main
+  Callback clb;
+  void* clb_context;
+
+  /// central lock, used for accesses to global tables except vars (order: 1)
+  LockT g_lock;
+
+  /// switch logging of read/write operations
+  bool log_flag = true;
+  bool final_output = (true && log_flag);
```

Comment:
When possible, rebase your branch with the new `master` so that the execution could support passing flags to `fasttrack` upon running drace, for e.g. `drrun.exe -c drace-client.dll -d fasttrack --stats --size --other-drace-flags [...]`. This feature has already been added to the `master` branch (after you branched off to make these changes). You could try it out to see if passing the flags to `fasttrack` works as you would have expected. In any case this would also take care of these merge conflicts along the way.

That way, you could revert the initial state of `log_flag` back to `false` so that the user can toggle whether or not to show the logs when a flag is passed (by default not showing them).
Consequently, any occurrences of `final_output` should be replaced with `log_flag` (especially when calling
`process_log_output();` on line 390)
```suggestion
  bool log_flag = false;
```